routes/items/itemsModel.js

```diff
@@ -2,25 +2,49 @@ const knex = require('knex');
 const db = require('../../data/dbConfig');
 
 module.exports = {
-    getAll,
-    getItemById,
-    updateItem,
-    deleteItem
-}
+  getAll,
+  getItemById,
+  updateItem,
+  deleteItem,
+  getItemByCategory,
+  getItemByCondition,
+  getItemByZipCode,
+  getItemByCity,
+};
 
 function getAll() {
-    return db('items');
+  return db('items');
 }
 
 function getItemById(id) {
-    return db('items').where({id}).first();
+  return db('items')
+    .where({ id })
+    .first();
 }
 
 async function updateItem(id, updatedItem) {
-    const updated = await getItemById(id).update(updatedItem);
-    return updated;
+  const updated = await getItemById(id).update(updatedItem);
+  return updated;
 }
 
 function deleteItem(id) {
-    return  db('items').where({id}).delete();
+  return db('items')
+    .where({ id })
+    .delete();
+}
+
+function getItemByCategory(category) {
+  return db('items').where(category);
+}
+
+function getItemByCondition(condition) {
+  return db('items').where(condition);
+}
+
+function getItemByZipCode(zipcode) {
+  return db('items').where(zipcode);
+}
+
+function getItemByCity(city) {
```

Comment:
Noticed in this function if a city was put in as lowercase or uppercase it returns two different sets of results.

Reply:
Taking a look

Reply:
This should be fixed now @erik-sandoval. Can you verify so I can merge the PR?